README.md

```diff
@@ -1,318 +1,41 @@
-README.md](exporter/README.md).
-
-```yaml
-exporters:
-  opencensus:
-    headers: {"X-test-header": "test-header"}
-    compression: "gzip"
-    cert_pem_file: "server-ca-public.pem" # optional to enable TLS
-    endpoint: "localhost:55678"
-    reconnection_delay: 2s
-
-  logging:
-    loglevel: debug
-
-  jaeger_grpc:
-    endpoint: "http://localhost:14250"
-
-  jaeger_thrift_http:
-    headers: {"X-test-header": "test-header"}
-    timeout: 5
-    url: "http://localhost:14268/api/traces"
-
-  zipkin:
-    url: "http://localhost:9411/api/v2/spans"
-
-  prometheus:
-    endpoint: "localhost:8889"
-    namespace: "default"
-```
-
-### <a name="config-extensions"></a>Extensions
-
-Extensions are provided to monitor the health of the OpenTelemetry
-Collector. Extensions are optional. By default, no extensions are configured.
-
-A basic example of all available extensions is provided below. For detailed
-extension configuration, please see the [extension
-README.md](extension/README.md).
-
-```yaml
-extensions:
-  health_check: {}
-  pprof: {}
-  zpages: {}
-```
-
-### <a name="config-service"></a>Service
-
-The service section is used to configure what features are enabled in the
-OpenTelemetry Collector based on the configuration found in the receivers,
-processors, exporters, and extensions sections. The service section
-consists of two sub-sections:
-
-* extensions
-* pipelines
-
-Extensions consist of a list of all extensions to enable. For example:
-
-```yaml
-    service:
-      extensions: [health_check, pprof, zpages]
-```
-
-Pipelines can be of two types:
-
-* metrics: collects and processes metrics data.
-* traces: collects and processes trace data.
-
-A pipeline consists of a set of receivers, processors, and exporters. Each
-receiver/processor/exporter must be specified in the configuration to be
-included in a pipeline and each receiver/processor/exporter can be used in more
-than one pipeline.
-
-*Note:* For processor(s) referenced in multiple pipelines, each pipeline will
-get a separate instance of that processor(s). This is in contrast to
-receiver(s)/exporter(s) referenced in multiple pipelines, where only one
-instance of a receiver/exporter is used for all pipelines.
-
-The following is an example pipeline configuration. For more information, refer
-to [pipeline documentation](docs/pipelines.md).
-
-```yaml
-service:
-  pipelines:
-    traces:
-      receivers: [opencensus, jaeger]
-      processors: [batch, queued_retry]
-      exporters: [opencensus, zipkin]
-```
-
-## Other Information
-
-### Extending the Collector
-
-The OpenTelemetry collector can be extended or embedded into other applications.
-
-The list of applications extending the collector:
-
-* [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
-* [jaeger-opentelemetry-collector](https://github.com/jaegertracing/jaeger-opentemenetry-collector)
-
-### Owners
+---
+
+<p align="center">
```

Comment:
Why do we use html in a markdown?

Reply:
Because markdown does not support everything needed (e.g. center). See: https://github.com/open-telemetry/opentelemetry-collector/blob/2b8085c369a550f0aea7dbf0928d7ce07ce1b960/README.md for desired result

Reply:
ok